Grammar/python.gram

```diff
@@ -311,18 +311,22 @@ block[asdl_stmt_seq*] (memo):
     | simple_stmts
     | invalid_block
 
+star_element[expr_ty]:
+    | '*' a=bitwise_or { _Py_Starred(a, Load, EXTRA) }
+    | '(' '*' a=bitwise_or ')' { _Py_Starred(a, Load, EXTRA) }
```

Comment:
This patch makes `(*x), y` valid, but with multiple layers of parentheses `(((*x))), y` is still considered invalid.